authorization/v2/uniformResourceIdentifier_test.go

```diff
@@ -224,3 +224,59 @@ func Test_UnmarshalJSON__should_return_correct_URI(t *testing.T) {
 
 	assert.Equal(t, identifier, expectedURI)
 }
+
+func Test_CompareURI__should_return_true_with_source_in_target_set(t *testing.T) {
+	tests := []struct {
+		name   string
+		source UniformResourceIdentifier
+		target []UniformResourceIdentifier
+	}{
+		{
+			name: "with valid source URI in target set, only path",
+			source: UniformResourceIdentifier{
+				path:      "hs:hs_auth:api:v2:provide_access_to_uri",
+				arguments: nil,
```

Comment:
These `nil` initializations are redundant but idk if you feel they improve readability feel free to leave them.

Reply:
Ah nice, I'll get rid of them 👍